ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/miner/MinerGetMinPriorityFee.java

```diff
@@ -36,6 +36,7 @@ public String getName() {
   @Override
   public JsonRpcResponse response(final JsonRpcRequestContext requestContext) {
     return new JsonRpcSuccessResponse(
-        requestContext.getRequest().getId(), miningParameters.getMinPriorityFeePerGas().getValue());
+        requestContext.getRequest().getId(),
+        miningParameters.getMinPriorityFeePerGas().toShortHexString());
```

Comment:
I see that other APIs (like `eth_getGasPrice`) use `Quantity:create` to format the values returned, so you may consider to use that for consistency

Reply:
done! Thanks!